Makefile

```diff
@@ -306,32 +347,30 @@ mod-download:
 	done
 
 .PHONY: acceptance.go-mod-vendor autoscaler.go-mod-vendor changelog.go-mod-vendor \
-				changeloglockcleander.go-mod-vendor
+				changeloglockcleaner.go-mod-vendor
 go-mod-vendor: clean-vendor acceptance.go-mod-vendor autoscaler.go-mod-vendor changelog.go-mod-vendor \
-							 changeloglockcleander.go-mod-vendor
+							 changeloglockcleaner.go-mod-vendor
 acceptance.go-mod-vendor:
-	make --directory='${go-acceptance-dir}' go-mod-vendor
+	make --directory='${acceptance-dir}' go-mod-vendor
 autoscaler.go-mod-vendor:
-	make --directory='${go-autoscaler-dir}' go-mod-vendor
+	make --directory='${autoscaler-dir}' go-mod-vendor
 changelog.go-mod-vendor:
-	make --directory='${go-changelog-dir}' go-mod-vendor
-changeloglockcleander.go-mod-vendor:
-	make --directory='${go-changeloglockcleander-dir}' go-mod-vendor
+	make --directory='${changelog-dir}' go-mod-vendor
+changeloglockcleaner.go-mod-vendor:
+	make --directory='${changeloglockcleaner-dir}' go-mod-vendor
 
-.PHONY: uuac
-uaac:
-	which uaac || gem install cf-uaac
 
 .PHONY: update-uaac-nix-package
 update-uaac-nix-package:
 	make --directory='./nix/packages/uaac' gemset.nix
 
 .PHONY: deploy-autoscaler deploy-register-cf deploy-autoscaler-bosh deploy-cleanup
-deploy-autoscaler: go-mod-vendor uaac db scheduler deploy-autoscaler-bosh
+deploy-autoscaler: deploy-autoscaler-bosh
```

Comment:
Here we propagate the dependencies one layer down because they all can be attributed to `deploy-autoscaler-bosh`.